src/components/Scrollable/Scrollable.tsx

```diff
@@ -130,6 +130,8 @@ export class Scrollable extends Component<ScrollableProps, State> {
             {...scrollable.props}
             {...rest}
             ref={this.setScrollArea}
+            // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
Adding a little comment here too say why we know better than eslint would be handy for the future